docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -188,10 +188,10 @@ bash-5.2$ diff -w .kcp/admin.kubeconfig test.yaml
 
 Following is an example of using the alternate kubeconfig file, in a
 context where the domain name "yep.yep" resolves to an IP address of
-the network namespace in which the kcp server is running.
+the network namespace in which the space provider's server is running.
 
 ```console
-bash-5.2$ KUBECONFIG=.kcp-yep.yep/admin.kubeconfig kubectl ws .
+bash-5.2$ KUBECONFIG=.cp-yep.yep/admin.kubeconfig kubectl ws .
```

Comment:
Oops, this was wrong to begin with. Instead of `.kcp-yep.yep/admin.kubeconfig` it should have said `test.yaml` (the file created the the command above).